src/mame/misc/orientp.cpp

```diff
@@ -0,0 +1,179 @@
+// license:BSD-3-Clause
+// copyright-holders:flama12333
+/*************************************************************************
+
+// Features Notes Was from archived websited:
+src: Chang yu website
```

Comment:
I think copying entire parts of a website in the source is questionable, but I'll defer to others on this